.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  run:
+    name: Node ${{ matrix.node }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [6, 8.16.1, 10, 12]
```

Comment:
Why such a specific version requirement for node 8, when all the other versions are just locked down to the major version number?

Reply:
There's a bug currently which on Windows it installs 8.10 which has an older npm that doesn't have `npm ci`

https://github.com/actions/setup-node/issues/27

Reply:
Cool, could you note that in a comment so the reasoning behind that version choice is clear and so we remember to remove the version restriction once that issue is resolved?